bench_pytorch/README.md

```diff
@@ -5,29 +5,55 @@
 The implementation of benchmarking [PyTorch](https://github.com/pytorch/pytorch) uses the [Transformers Library by Huggingface](https://github.com/huggingface/transformers) under the hood. The reason being, Transformers provides an easy to use interface for Llama-2-7B model in PyTorch backend.
 
 
-### 🚀 Running the PyTorch Benchmark.
+## 🚀 Running the PyTorch Benchmark.
 
-You can run the PyTorch benchmark using the following command:
+We can run the PyTorch benchmark for two models: [Llama2](https://huggingface.co/meta-llama/Llama-2-7b) and [Mistral-7B v0.1](https://huggingface.co/mistralai/Mistral-7B-v0.1) Here is how we run benchmark for PyTorch.
```

Comment:
are these models correct or are the chat models?

Reply:
Let me recheck this ones